packages/web/src/components/spa/HackAUBG/AboutHackathon/AboutHackathon.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './about_hackathon.css';
+
+export const AboutHackathon = () => {
+    return (
+        <div className="about-container-hack-aubg" id="AboutSection">
+            <h1 className="about-title-hack-aubg">ABOUT</h1>
+            <div className="about-text-container-hack-aubg">
+                <p className="about-text-hack-aubg">
+                    HackAUBG is a 3 day-long programming hackathon organized by
```

Comment:
Font is too small. Refer to the current [HackAUBG page](https://thehub-aubg.com/hackaubg):
![image](https://user-images.githubusercontent.com/104720011/216776606-efd17a5a-40b8-4d4d-83fa-ce9e0cddd95c.png)

NB: Probably you should break down the text in more paragraphs, bring it a bit towards the center and use an even larger font than the one used in the hub-aubg.com/hackaubg